tchannel/context.py

```diff
@@ -73,12 +73,25 @@ def __exit__(self, type, value, traceback):
         _LOCAL.context = self._old_context
 
 
-def get_current_context():
-    """
-    :return: The current :py:class:`RequestContext` for this thread.
-    """
-    return _LOCAL.context
-
-
-def request_context(parent_tracing):
-    return StackContext(lambda: RequestContext(parent_tracing))
+# noinspection PyMethodMayBeStatic
```

Comment:
@abhinav @blampe any concerns that I am removing these static methods?  I made sure tchannel code does not call them anymore, it always uses context provider instance.